src/server/svr_comm.c

```diff
@@ -271,15 +271,15 @@ send_nodestat_req(void)
 
 	update_msvr_stat(1, CACHE_MISS);
 
+	if (mtfd == -1)
+		return;
```

Comment:
shouldn't this be done before the call to update_msvr_stat() ?

Reply:
update_msvr_stat() is updating a potential cache miss. that has to be done anyway except when it is initially invoked. I'll add the changes for that.